### ISSUE TITLE
[Error: Attempt to invoke virtual method 'int java.lang.Object.hashCode()' on a null object reference]

### ISSUE THREAD

**Issue by zsofitoth** [FormidableLabs/react-native-app-auth#476]
## Issue

```JavaScript

const SPOTIFY_CONFIG = {
  clientId: '<client_id>',
  clientSecret: '<client_secret>',
  redirectUrl: 'com.app:/oauth',
  scope: ['user-follow-read'],
  serviceConfiguration: {
    authorizationEndpoint: 'https://accounts.spotify.com/authorize',
    tokenEndpoint: 'https://accounts.spotify.com/api/token',
  },
}

const authState = await authorize(SPOTIFY_CONFIG)
``` 

The code above throws the following error:
```[Error: Attempt to invoke virtual method 'int java.lang.Object.hashCode()' on a null object reference]```

According to the documentation that’s the right way to set the config object for Spotify. Also, it works on iOS like that.

Managed to fix it with the workaround where added empty **issuer** field:
```JavaScript

const SPOTIFY_CONFIG = {
  issuer: '',
  clientId: '<client_id>',
  clientSecret: '<client_secret>',
  redirectUrl: 'com.app:/oauth',
  scope: ['user-follow-read'],
  serviceConfiguration: {
    authorizationEndpoint: 'https://accounts.spotify.com/authorize',
    tokenEndpoint: 'https://accounts.spotify.com/api/token',
  },
}

const authState = await authorize(SPOTIFY_CONFIG)
``` 
---

## Environment

* **Your Identity Provider**: Spotify
* **Platform that you're experiencing the issue on**: Android
* **Are you using Expo?** No

**Comment by aeirola** [comment]
This seems to be a duplicate of #473 which was fixed in version 5.1.1

**Comment by Jay-A-McBee** [comment]
Closing - fixed by https://github.com/FormidableLabs/react-native-app-auth/pull/475
